docs/execution-analysis/simulation.md

```diff
@@ -1,6 +1,6 @@
 # Simulation
 
-During development, the speed of homomorphic execution can be a blocker for fast prototyping. You could call the function you're trying to compile directly, of course, but it won't be exactly the same as FHE execution, which has a certain probability of error (see [Exactness](../core-features/table\_lookups.md#table-lookup-exactness)).
+During development, the speed of homomorphic execution can be a blocker for fast prototyping. You could call the function you're trying to compile directly, of course, but it won't be exactly the same as FHE execution, which has a certain probability of error (see [Exactness](../core-features/table_lookups.md#table-lookup-exactness)).
```

Comment:
This doc
```suggestion
This document explains how to use the simulation feature to speedup prototyping and detect overflows during Fully Homomorphic Encryption (FHE) computation. 

Simulation allows to mimic the behavior of FHE execution without actually performing the encrypted computations.
```

Reply:
This original sentence is not very clear to me:
" You could call the function you're trying to compile directly, of course, but **it** won't be exactly the same as FHE execution, **which** has a certain probability of error "

- Is this sentence describing the simulation feature? or the issue to be addressed by the simulation feature?
- What does "it" refers to - the simulated function?
- What doesn "which" refers to - the FHE function? of the simulation process?

I would suggest rewriting this part with better clarity. 
For example, 

> Simulations allow to.... (what does it do)  It is helpful to .....(why is it useful)